lib/auth/tls_test.go

```diff
@@ -1377,6 +1377,12 @@ func TestWebSessionMultiAccessRequests(t *testing.T) {
 	err = clt.CreateAccessRequest(ctx, roleReq)
 	require.NoError(t, err)
 
+	// Create remote cluster
+	rc, err := types.NewRemoteCluster("foobar")
+	require.NoError(t, err)
+	err = tt.server.AuthServer.AuthServer.CreateRemoteCluster(rc)
+	require.NoError(t, err)
```

Comment:
Like @GavinFrazar mentioned, it's not very clear what creating a "foobar" remote cluster is accomplishing w/r/t testing invalid cluster names in access requests.

Reply:
Have added more tests for the cluster name validation and added a comment to the change in that existing change to explain why it was changed.